misc/check-ts.ts

```diff
@@ -0,0 +1,35 @@
+import {join} from 'path';
+import {ChildProcess, spawn, SpawnOptions} from 'child_process';
+
+const options: SpawnOptions = {
+  cwd: join(__dirname, '..')
+};
+
+const tsConfigs = [
+  'src/tsconfig.json', 'src/tsconfig.spec.json', 'demo/tsconfig.json', 'e2e-app/tsconfig.json',
+  'e2e-app/tsconfig.spec.json'
+];
+
+const args = ['--noEmit', '--noUnusedLocals'];
+
+const messages = new Set<string>();
+const running = new Set<ChildProcess>();
+
+tsConfigs.forEach(tsConfig => {
+  const child = spawn('npx', ['tsc', ...args, `--project`, tsConfig], options);
```

Comment:
Could you please add error handling?
i.e.
`.on('error', function( err ){ throw err })`
as mentioned in https://stackoverflow.com/questions/27688804/how-do-i-debug-error-spawn-enoent-on-node-js

This might be useful when it doesn't work on a local environment.

Also this file is added to avoid adding `noUnusedLocals `and `noUnusedParameters `in .tsconfig.json, right?

Reply:
@gpolychronisAmadeus good idea, will update it.

> Also this file is added to avoid adding noUnusedLocals and noUnusedParameters in .tsconfig.json, right?

Yes, it's the one that is executed with `yarn check-ts`

Reply:
@maxokorokov 
As discussed:

- In windows environment it doesn't work. Please add `shell: true` in options.
- Also please reset the color.

Reply:
@gpolychronisAmadeus thanks for checking it on Windows, fixed both in the update.